### PR TITLE
Feature 22224 sp blogpage

### PR DESCRIPTION
**Pull request by shibasundar-marmeto**
Pull Request (PR) Request Template:

### Problem
 blog listing page and articlePage

Monday Card Link: [https://marmeto-force.monday.com/boards/5752050817?userId=54242963]
preview blog list : [ https://parkavenue-india.myshopify.com/blogs/all-blogs?preview_theme_id=165576769816]

preview blog desc : [ https://parkavenue-india.myshopify.com/blogs/all-blogs/lorem-ipusm-lorem-ipusm-dor-lorem-ipusm-lorem-ipusm-10?preview_theme_id=165576769816]

password : marmeto

### Solution
blog page and article page created  

### Impacted Areas
blog page and article page

### Done Checklist
- [x] Performed unit testing to ensure functionality and identify potential issues.
- [x] Followed basic code standards and best practices. (indentation, media queries, basic customizer things)
- [x] Added comments to explain complex logic or improve code readability.
- [x] Followed Naming conventions and proper variable declaration.
- [x] Handled edge cases and errors gracefully to prevent crashes.
- [x] Optimized the code for production to ensure performance.

Please review this PR and provide feedback. Thank you!

### CODE REVIEW

File: assets/custom-article.css [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+.related-article-title a {
+  text-decoration: none;
+  color: #0e0e10;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 140%;
+  letter-spacing: -0.2px;
+}
+.article-desc {
+  color: #231f20;
```
Comment (hatprabMarmeto) [inline assets/custom-article.css:94]: A good practice is to have to color define globally for the text so it save CSS file length. And where we need to override/change the color to something different there only we should be changing it.

File: assets/custom-article.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 160%;
+  letter-spacing: 1px;
+}
+.article-content-container p {
+  color: #231f20;
```
Comment (hatprabMarmeto) [inline assets/custom-article.css:101]: Make color Globally for all the common tags such as 
h1,h2,p, a , etc .

File: assets/custom-blog.css [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  gap: 1.6rem;
+}
+
+.item:nth-child(4) {
+  grid-area: 3 / 2 / 4 / 4;
+  display: flex;
+  gap: 1.6rem;
+}
+
+.recent-article-image-container img {
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:65]: Image are getting stretched please fix it.

File: assets/custom-blog.css [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+.recentpost-container .item:nth-child(3) .recent-article-image-container {
+  width: 42.7%;
+  max-height: 20.6rem;
+}
+
+.recentpost-container .item:nth-child(4) .recent-article-image-container {
+  width: 42.7%;
+  max-height: 20.6rem;
+}
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:88]: All of these are common and can be grouped together

File: assets/custom-blog.css [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+
+.main-blog-title {
+  display: flex;
+  gap: 2.8rem;
+  text-decoration: none;
+  color: #0e0e10;
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 140%;
+  letter-spacing: -0.2px;
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:112]: Font Should have been picked from global class

File: assets/custom-blog.css [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+  display: flex;
+  justify-content: space-between;
+  color: #0e0e10;
+}
+.article-desc {
+  color: #231f20;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 160%;
+  letter-spacing: 1px;
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:155]: Should have been a common class that we could use multiple places

File: assets/custom-blog.css [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+
+  .recentpost-container {
+    padding: 0;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+  }
+
+  .item:nth-child(n) {
+    grid-area: auto;
+  }
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:179]: What is the purpose of
`.item:nth-child(n)`

File: assets/custom-blog.css [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+    padding: 0;
+  }
+
+  .main-blog-container {
+    margin-top: 4.8rem;
+  }
+
+  .recent-post-main {
+    padding: 0 1.7rem 0 1.7rem;
+  }
```
Comment (hatprabMarmeto) [inline assets/custom-blog.css:210]: Suggestion: the padding can be written in short hand 
`padding: 0 1.7rem 0 1.7rem; `
can be written as 
`padding: 0 1.7rem;`

File: sections/main-blog.liquid [hunk 9]
```diff
@@ -1,6 +1,7 @@
 {{ 'component-article-card.css' | asset_url | stylesheet_tag }}
 {{ 'component-card.css' | asset_url | stylesheet_tag }}
```
Comment (hatprabMarmeto) [inline sections/main-blog.liquid:2]: Are this required ?

File: sections/main-blog.liquid [hunk 10]
```diff
@@ -1,6 +1,7 @@
 {{ 'component-article-card.css' | asset_url | stylesheet_tag }}
```
Comment (hatprabMarmeto) [inline sections/main-blog.liquid:1]: Is this required ?

**Review by hatprabMarmeto — CHANGES_REQUESTED** [review]
@shibasundar-marmeto 
Please check the comments 

Also please fix the fonts and design as it does't match what is in the figma.

File: assets/custom-article.css (hunk 2 above)
Comment (shibasundar-marmeto) [reply assets/custom-article.css:108]: this color is for only content inside articles it is used only one time

File: assets/custom-blog.css (hunk 4 above)
Comment (shibasundar-marmeto) [reply assets/custom-blog.css:99]: fixed

File: assets/custom-blog.css (hunk 5 above)
Comment (shibasundar-marmeto) [reply assets/custom-blog.css]: fixed

File: assets/custom-blog.css (hunk 8 above)
Comment (shibasundar-marmeto) [reply assets/custom-blog.css:249]: fixed

**Comment by S-Darshan** [comment]
Closing PR reason: new PR will be opened with UT changes.

**Pull request closed without merging**